lib/bigcommerce/resources/customers/customer.rb

```diff
@@ -26,5 +29,21 @@ class Customer < Resource
     def self.count(params = {})
       get 'customers/count', params
     end
+
+    # Generate a token that can be used to log the customer into the storefront.
+    # This requires your app to have the store_v2_customers_login scope and to
+    # be installed in the store.
+    def login_token(config = Bigcommerce.config)
```

Comment:
Thoughts on using keyword args here so the api is similar to the other hash param methods? just change the `=` to `:`